west.yml

```diff
@@ -277,12 +279,13 @@ manifest:
       revision: 2b498e6f36d6b82ae1da12c8b7742e318624ecf5
       path: modules/lib/gui/lvgl
     - name: mbedtls
-      revision: 6ec4abdcda78dfc47315af568f93e5ad4398dea0
+      revision: 3217c450180fd5e817601c6f479116de69e57461
       path: modules/crypto/mbedtls
       groups:
         - crypto
     - name: mcuboot
-      revision: 78bfe750cde0f8dfcf81bb4c0a68243d45906def
+      remote: mcu-tools
+      revision: d4394c2f9b76e0a7b758441cea3a8ceb896f66c8
```

Comment:
Is my understanding correct that this implies that it is no longer relying on this mirror: https://github.com/zephyrproject-rtos/mcuboot/ ?

I don't see this change advertised in the commit message. Would it be worth adding a more visible note about this change?

Reply:
This is a temporary change to be able to pull the changes I made to upstream MCUboot. Zephyr's fork of it hasn't been updated yet, so the upstream remote needs to be used. (The change from `pull/[...]` to the hash is because my PR got merged upstream).
This will be updated once Zephyr's fork of MCUboot is updated, which will happen before this PR can be merged.